recipes/libnfs/all/conanfile.py

```diff
@@ -66,6 +76,9 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "libnfs")
         self.cpp_info.set_property("cmake_target_name", "libnfs::libnfs")
         self.cpp_info.set_property("pkg_config_name", "libnfs")
-        self.cpp_info.libs = ["nfs"]
+        if Version(self.version) >= "6.0.2" and self.settings.os == "Windows":
+            self.cpp_info.libs = ["libnfs"]
+        else:
+            self.cpp_info.libs = ["nfs"]
```

Comment:
```suggestion
         self.cpp_info.libs = ["nfs"]
```

Hello! It works on Windows, using version 2.0.6 with the patch, I just tested locally running Conan 2.14:

- [libnfs-6.0.2-windows-static.log](https://github.com/user-attachments/files/19485313/libnfs-6.0.2-windows-static.log)
- [libnfs-6.0.2-windows-shared.log](https://github.com/user-attachments/files/19485323/libnfs-6.0.2-windows-shared.log)

Reply:
@toge Thank you for informing. I'll re-check, because it really worked for me.

Reply:
I think @uilianries  - you may have meant building _without_ the patch?